src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -16,7 +16,7 @@
 
     public static final String COMMAND_WORD = "find";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain any of "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names or tags contain any of "
             + "the specified keywords (case-sensitive) and displays them as a list with index numbers.\n"
             + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
             + "Example: " + COMMAND_WORD + " alice bob charlie";
```

Comment:
It is fine since you have already updated the `MESSAGE_USAGE`, but it would be even better if you updated the example in the `MESSAGE_USAGE` too to include tags